src/equations/lattice_boltzmann_2d.jl

```diff
@@ -353,6 +353,12 @@ end
 end
 
 
+# TODO: flux_lax_friedrichs is not of the usual form for this equation???
+# Calculate maximum wave speed for local Lax-Friedrichs-type dissipation
+# @inline function max_abs_speed_naive(u_ll, u_rr, orientation, equations::LatticeBoltzmannEquations2D)
+#   λ_max =
+# end
+
```

Comment:
@sloede The `flux_lax_friedrichs` is not of the usual form of a local Lax-Friedrichs flux for these equations since it uses
```julia
abs.(v_alpha) .* (u_rr - u_ll)
```
instead of 
```julia
maximum(abs, v_alpha) * (u_rr - u_ll)
```
How shall we handle this? Isn't this kind of numerical flux more like a `flux_godunov`?